werkomgeving/gegeven/index.md

```diff
@@ -5,8 +5,8 @@
 Een gegeven is *een vastgelegde uitspraak over een getypeerde eigenschap van een object*. Kenmerkend aan een gegeven zijn een aantal elementen:
 
 1. Een gegeven is **vastgelegd**: op papier, in steen, in een digitale vorm. In elk van deze vormen van sprake zijn van meer of minder mate van vluchtigheid of persistentie: gegevens in een bericht zullen minder persistent zijn dan gegevens die duurzaam worden opgeslagen in een administratie.
-2. Een gegeven is een **uitspraak**: zo'n uitspraak kan een bewering, stelling of waarneming zijn en zo'n uitspraak kan gedaan worden door een mens of een sensor. Ook kan zo'n uitspraak betrekking hebben op een berekening of afleiding.
-3. Een gegeven gaat over een **object**: het gaat over *iets*: de objecten die relevant zijn in het domein waarover we gegevens verwerken.
+2. Een gegeven gaat over een **object**: het gaat over *iets*: de objecten die relevant zijn in het domein waarover we gegevens verwerken.
+3. Een gegeven is een **uitspraak**: zo'n uitspraak kan een bewering, stelling of waarneming zijn en zo'n uitspraak kan gedaan worden door een mens of een sensor. Ook kan zo'n uitspraak betrekking hebben op een berekening of afleiding.
```

Comment:
> Een gegeven is een **uitspraak**

Voor mijn gevoel klopt dit niet. Een uitspraak is n.m.m. zelf per definitie niet vastgelegd.  Een gegeven kan wel de VASTLEGGING van een uitspraak zijn alhoewel de term ‘uitspraak’ hierbij nog niet helemaal correct aanvoelt. De definitie van ‘uitspraak’ is wel al aardig correct maar de term bij die definitie (‘uitspraak’) dus nog niet. Een sensor doet nl. geen uitspraak maar wel en waarneming.

Andere opties zijn ‘statement’, ‘uiting’ al voelen die ook niet helemaal 100%.

Reply:
Ik snap dat het niet letterlijk moet worden opgevat. Ik denk dat ik de term ‘Uitdrukking’ ook beter vind en in dat geval doet mijn opmerking m.b.t. vastlegging er ook niet meer zo toe omdat uitdrukken ook kan door vast te leggen.